src/main/webapp/app/course/manage/course-update.component.html

```diff
@@ -47,6 +47,22 @@ <h4 *ngIf="course.id" id="jhi-course-heading-edit" jhiTranslate="artemisApp.cour
                         </div>
                     </ng-container>
                 </div>
+                <div class="form-group" *ngIf="course && course.maxComplaints > 0">
+                    <label class="form-control-label" jhiTranslate="artemisApp.course.maxComplaints" for="field_maxComplaints">Maximum Number of Complaints</label>
+                    <input required type="number" class="form-control" name="maxComplaints" id="field_maxComplaints" formControlName="maxComplaints" />
+                </div>
+                <div class="form-group">
+                    <label class="form-control-label" jhiTranslate="artemisApp.course.maxComplaintTimeDays" for="field_maxComplaintTimeDays"
+                        >Deadline for complaints in days after result date</label
```

Comment:
please add some info buttons / question mark buttons with an explanation what this means in natural language (German and English)

Also mention the logic with the assessment due date